src/Intersection.jl

```diff
@@ -423,16 +423,21 @@ This method relies on having available the `constraints_list` of the polytope
 This method of overapproximation can return a non-empty set even if the original
 intersection is empty.
 """
+function ρ(d::AbstractVector{N},
+           cap::Intersection{N, S1, S2};
+           kwargs...) where {N<:Real, S1<:HPolyhedron{N}, S2<:AbstractPolytope{N}}
```

Comment:
I do not understand why this method is needed. It should be covered by the method below with `{N<:Real, S1<:LazySet{N}, S2<:AbstractPolytope{N}}`. At least the type `S1<:HPolyhedron{N}` should be generalized to `S1<:AbstractPolyhedron{N}`.